baggers.js

```diff
@@ -12950,12 +12950,12 @@ var data = {
       "picture": "https://www.gravatar.com/avatar/63d261113651caa0dc887445c61ea48a?s=200",
       "websites": [
         {
-          "title": "Blog Xebia",
-          "href": "http://blog.xebia.fr/category/mobilite"
+          "name": "Blog Xebia",
```

Comment:
Please use only one of the list here: https://github.com/brownbaglunch/bblfr_data#websites

I'd recommend `Web`.